packages/ui/src/components/Table/STable.vue

```diff
@@ -410,20 +454,20 @@ function getStyleOrClass<T extends object | string, S>(prop: T | ((args: S) => T
   return prop || undefined
 }
 
-function isCheckBoxDisabled(column: TableActionColumnApi, row: TableRow, index: number) {
-  return !column.selectable || column.selectable(row, index)
+function isRowSelectable(row: TableRow, index: number) {
+  return !!(
+    activeSelectionColumn?.value &&
+    (!activeSelectionColumn.value.selectable || activeSelectionColumn.value.selectable(row, index))
+  )
 }
 
-function getDefaultCellValue(row: TableRow, column: TableColumnApi, index: number) {
-  return column.formatter ? column.formatter(row, column, get(row, column.prop), index) : get(row, column.prop)
-}
-
-function getCellTooltipContent(row: TableRow, column: TableColumnApi | TableActionColumnApi) {
-  if (!column.showOverflowTooltip || !isDefaultColumn(column)) {
-    return
+function getTemplateRowKey(row: TableRow, rowIndex: number): number | string | symbol {
+  const res = props.rowKey ? rowKeys.get(row) : rowIndex
+  if (typeof res === 'string' || typeof res === 'number' || typeof res === 'symbol') {
+    return res
   }
 
-  return String(get(row, column.prop))
+  return JSON.stringify(res)
```

Comment:
Why do you use JSON here? Also, if you return a string, why your function signature returns `number | string | symbol`?

Reply:
There are two return statements. One for `number | string | symbol` values and JSON for others, like objects.